operator/gitops/argocd/pipeline-service/openshift-pipelines/config-pac.yaml

```diff
@@ -0,0 +1,13 @@
+---
+# we need the pac config map in the gitops tree so that we can then patch it with RHTAP specific updates
+# previously, when pipeline-service deployed pac via the upstream yaml, that is how the config map got into
+# the gitops tree, but now that we are allowing openshift-pipelines operator install pac, we need to create
+# this empty config map
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: pipelines-as-code
+  namespace: openshift-pipelines
+data:
+  # The application name, you can customize this label. If using the Github App you will need to customize the label on the github app setting as well.
+  application-name: "Pipelines as Code CI"
```

Comment:
the RHTAP patch is a gitops / kustomize replace, so I wanted to make sure the entry it is replacing exists so there are no issues there